index.js

```diff
@@ -480,33 +494,22 @@ RedisClient.prototype.connection_gone = function (why) {
     }, this.retry_delay);
 };
 
-RedisClient.prototype.on_data = function (data) {
-    // The data.toString() has a significant impact on big chunks and therefor this should only be used if necessary
-    // debug("Net read " + this.address + " id " + this.connection_id + ": " + data.toString());
-
-    try {
-        this.reply_parser.execute(data);
```

Comment:
We do not need this code anymore as the parser won't throw any non parser errors anymore.